behaviours/meme_rater.py

```diff
@@ -28,25 +31,29 @@
 IMG_FILE_EXTENSIONS: Final = {"jpg", "jpeg", "png", "webp"}
 
 
-async def get_meme_rating(image_url: str, user: str):
+async def get_meme_rating(image_url: str, user: str) -> MemeAnswer:
     image = requests.get(image_url, stream=True)
     if not image.raw:
         logging.info("Not image")
         return ""
     try:
         kitty_reasoner_meme_rater = commons.agents.agent("reasoner_meme_rater")
         if kitty_reasoner_meme_rater:
-            response = await kitty_reasoner_meme_rater.run(image=image, user=user)
+            response: MemeAnswer = await kitty_reasoner_meme_rater.run(
+                image=image, user=user
+            )
         else:
             raise Exception("No reasoner meme rater")
     except Exception as e:
         logging.info(f"Error running reasoner meme rater: {e}")
         response = await commons.agents.agent("meme_rater").run(image=image, user=user)
     logging.info(f"Meme rating response: {response}")
     try:
-        return min(max(0, int(response)), 10)
+        return MemeAnswer(
+            rate=min(max(0, int(response.rate)), 10), explanation=response.explanation
+        )
     except Exception as e:
-        return 0
+        return MemeAnswer(rate=0, explanation="SHIT BROKE")
```

Comment:
Is there any particular reason we need to catch this exception / return any value here?

Reply:
There was a zero before with it only returning a rating so probably not but I didn't study it too closely.